_src/static/sass/base/_reset.scss

```diff
@@ -37,18 +37,13 @@ video {
 body {
     color: $text-base-color;
     background: $view-bg;
-    font-size: $font-scale-medium;
+    font-size: font-size(base);
     font-family: $font-family-sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-.hd-1,
-.hd-2,
-.hd-3,
-.hd-4,
-.hd-5,
-.hd-6 {
-    @extend %reset-headings;
+p, ol, ul, dl {
```

Comment:
NOTE: adding basic margin reset (using only bottom margin vs. normalize's top/bottom) for elements (see http://csswizardry.com/2012/06/single-direction-margin-declarations/)